src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.WebSearch/Properties/Resources.resx

```diff
@@ -129,6 +129,9 @@
   <data name="plugin_in_browser" xml:space="preserve">
     <value>In the default browser</value>
   </data>
+  <data name="plugin_in_browser_name" xml:space="preserve">
+    <value>In {0}</value>
+  </data>
```

Comment:
Comment `Like "Search the web in ..."` is missing. This helps translation system to understand it. 😅

Reply:
Hmm... Valid point. 🤣